messaging/src/main/java/org/axonframework/messaging/RemoteExceptionDescription.java

```diff
@@ -46,7 +48,13 @@
      * @return a {@link List} of {@link String} describing the given {@link Exception}
      */
     public static RemoteExceptionDescription describing(Throwable exception) {
-        return new RemoteExceptionDescription(createDescription(exception, new ArrayList<>()));
+        final boolean isPersistent = isExplicitlyNonTransient(exception);
+        return new RemoteExceptionDescription(createDescription(exception, new ArrayList<>()), isPersistent);
+    }
+
+    private static boolean isExplicitlyNonTransient(Throwable exception) {
```

Comment:
Kinda late, but what do you think about having this method in the `ExceptionUtils`? That would at least mean we have a single place for this static method, instead of requiring it in several projects :-[